content/en/docs/ambient/usage/extend-waypoint-wasm/wasm-module-distribution/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Distributing WebAssembly Modules
```

Comment:
i think the folder structure here is wrong.. you do not need wasm-module-distribution directory anymore. and adjust the weight below so that this comes after waypoint document